plugins/@grouparoo/marketo/src/lib/test.ts

```diff
@@ -0,0 +1,13 @@
+import { TestPluginMethod } from "@grouparoo/core";
+import { connect } from "./connect";
+
+export const test: TestPluginMethod = async ({ appOptions }) => {
+  try {
+    const client = await connect(appOptions);
+    const response = await client.lead.describe();
+    const success = response.success ? true : false;
+    return { success };
```

Comment:
Is there a message you could return here?  Perhaps something about the account?

Reply:
I had trouble finding something useful. I'll make a task.